.github/dependabot.yaml

```diff
@@ -3,10 +3,12 @@ version: 2
 enable-beta-ecosystems: true
 
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
+  - package-ecosystem: github-actions
+    directory: /
     schedule:
-      interval: "weekly"
+      interval: monthly
```

Comment:
Why not daily?

Reply:
This keeps us up to date but reduces the number of PRs we receive (we do get any security related updates out of band w/ this frequency).